lib/omnibus/packagers/rpm.rb

```diff
@@ -336,6 +337,15 @@ def create_rpm_file
     end
 
     #
+    #  Mark directories with the %dir directive.
+    #
+    def dir?(path)
+      full_path = build_dir + path.gsub('[%]','%')
+      return "%dir #{path}" if !File.symlink?(full_path) && File.directory?(full_path)
+      path
```

Comment:
Perhaps we can skip this helper and just add the logic to the template. That would match what we did for the config files:
https://github.com/chef/omnibus/blob/yzl/rhel7/resources/rpm/spec.erb#L65